lib/cstruct.mli

```diff
@@ -501,3 +501,6 @@ val append: t -> t -> t
 val concat: t list -> t
 (** [concat ts] is the concatenation of all the [ts]. It is not guaranteed that
  * the result is a newly created [t] in the zero- and one-element cases. *)
+
+val rev: t -> t
+(** [rev t] is [t] in reverse order. *)
```

Comment:
perhaps clarify that this *allocates* a fresh [t] and does not mutate the input

Reply:
Good idea! Done.